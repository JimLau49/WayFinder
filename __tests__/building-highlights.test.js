import React from "react";
import BuildingHighlights from "../components/building-highlights/building-highlights.component";
import renderer from "react-test-renderer";

// Kind: Snapshot + Unit Tests
// [US-2] Checks that all the highlights are rendering at their proper coordinates (latitude and longitude) also with proper styling
describe("Building Highlight Snapshop", () => {
  test("renders correctly", () => {
    const tree = renderer.create(<BuildingHighlights />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
