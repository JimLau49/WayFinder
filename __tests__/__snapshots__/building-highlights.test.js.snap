// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Building Highlight Snapshop renders correctly 1`] = `
<View>
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.49771,
          "longitude": -73.579034,
        },
        Object {
          "latitude": 45.497373,
          "longitude": -73.578339,
        },
        Object {
          "latitude": 45.496832,
          "longitude": -73.57885,
        },
        Object {
          "latitude": 45.497164,
          "longitude": -73.579543,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.496708,
          "longitude": -73.578525,
        },
        Object {
          "latitude": 45.497291,
          "longitude": -73.578017,
        },
        Object {
          "latitude": 45.496896,
          "longitude": -73.577102,
        },
        Object {
          "latitude": 45.496255,
          "longitude": -73.5777,
        },
        Object {
          "latitude": 45.496708,
          "longitude": -73.578525,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.492616,
          "longitude": -73.576544,
        },
        Object {
          "latitude": 45.493853,
          "longitude": -73.575514,
        },
        Object {
          "latitude": 45.494425,
          "longitude": -73.577054,
        },
        Object {
          "latitude": 45.493429,
          "longitude": -73.578126,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.49511,
          "longitude": -73.577861,
        },
        Object {
          "latitude": 45.495465,
          "longitude": -73.577501,
        },
        Object {
          "latitude": 45.49599,
          "longitude": -73.57849,
        },
        Object {
          "latitude": 45.495596,
          "longitude": -73.5788,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.495195,
          "longitude": -73.578535,
        },
        Object {
          "latitude": 45.494988,
          "longitude": -73.57876,
        },
        Object {
          "latitude": 45.495323,
          "longitude": -73.579401,
        },
        Object {
          "latitude": 45.495523,
          "longitude": -73.579196,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.493626,
          "longitude": -73.578718,
        },
        Object {
          "latitude": 45.493817,
          "longitude": -73.579072,
        },
        Object {
          "latitude": 45.494686,
          "longitude": -73.578034,
        },
        Object {
          "latitude": 45.494457,
          "longitude": -73.577636,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.496421,
          "longitude": -73.580935,
        },
        Object {
          "latitude": 45.4966,
          "longitude": -73.581461,
        },
        Object {
          "latitude": 45.496784,
          "longitude": -73.581303,
        },
        Object {
          "latitude": 45.496566,
          "longitude": -73.580804,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.497087,
          "longitude": -73.579927,
        },
        Object {
          "latitude": 45.496675,
          "longitude": -73.578983,
        },
        Object {
          "latitude": 45.496545,
          "longitude": -73.579115,
        },
        Object {
          "latitude": 45.49698,
          "longitude": -73.580014,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.495057,
          "longitude": -73.57828,
        },
        Object {
          "latitude": 45.494978,
          "longitude": -73.578139,
        },
        Object {
          "latitude": 45.494943,
          "longitude": -73.578176,
        },
        Object {
          "latitude": 45.495023,
          "longitude": -73.578332,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.496192,
          "longitude": -73.573781,
        },
        Object {
          "latitude": 45.495667,
          "longitude": -73.574304,
        },
        Object {
          "latitude": 45.49541,
          "longitude": -73.573748,
        },
        Object {
          "latitude": 45.495665,
          "longitude": -73.573482,
        },
        Object {
          "latitude": 45.495813,
          "longitude": -73.573807,
        },
        Object {
          "latitude": 45.496079,
          "longitude": -73.573544,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.496446,
          "longitude": -73.586218,
        },
        Object {
          "latitude": 45.496683,
          "longitude": -73.586074,
        },
        Object {
          "latitude": 45.496506,
          "longitude": -73.58569,
        },
        Object {
          "latitude": 45.49646,
          "longitude": -73.585711,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.458468,
          "longitude": -73.64132,
        },
        Object {
          "latitude": 45.458318,
          "longitude": -73.640863,
        },
        Object {
          "latitude": 45.457528,
          "longitude": -73.641468,
        },
        Object {
          "latitude": 45.457201,
          "longitude": -73.640653,
        },
        Object {
          "latitude": 45.456963,
          "longitude": -73.640853,
        },
        Object {
          "latitude": 45.457448,
          "longitude": -73.642094,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.460082,
          "longitude": -73.640892,
        },
        Object {
          "latitude": 45.459977,
          "longitude": -73.640967,
        },
        Object {
          "latitude": 45.45994,
          "longitude": -73.640865,
        },
        Object {
          "latitude": 45.460041,
          "longitude": -73.640774,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.459109,
          "longitude": -73.637828,
        },
        Object {
          "latitude": 45.458608,
          "longitude": -73.638351,
        },
        Object {
          "latitude": 45.459036,
          "longitude": -73.639505,
        },
        Object {
          "latitude": 45.459525,
          "longitude": -73.639165,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.457831,
          "longitude": -73.640486,
        },
        Object {
          "latitude": 45.457477,
          "longitude": -73.639675,
        },
        Object {
          "latitude": 45.457069,
          "longitude": -73.640057,
        },
        Object {
          "latitude": 45.45737,
          "longitude": -73.640879,
        },
        Object {
          "latitude": 45.457858,
          "longitude": -73.640548,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.458218,
          "longitude": -73.639895,
        },
        Object {
          "latitude": 45.45808,
          "longitude": -73.640005,
        },
        Object {
          "latitude": 45.458381,
          "longitude": -73.640789,
        },
        Object {
          "latitude": 45.458521,
          "longitude": -73.640679,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.457902,
          "longitude": -73.640122,
        },
        Object {
          "latitude": 45.458382,
          "longitude": -73.639744,
        },
        Object {
          "latitude": 45.458262,
          "longitude": -73.63945,
        },
        Object {
          "latitude": 45.457792,
          "longitude": -73.639825,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.459634,
          "longitude": -73.639192,
        },
        Object {
          "latitude": 45.459285,
          "longitude": -73.639457,
        },
        Object {
          "latitude": 45.459709,
          "longitude": -73.640322,
        },
        Object {
          "latitude": 45.459985,
          "longitude": -73.640136,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.456394,
          "longitude": -73.637372,
        },
        Object {
          "latitude": 45.456727,
          "longitude": -73.637101,
        },
        Object {
          "latitude": 45.456727,
          "longitude": -73.637101,
        },
        Object {
          "latitude": 45.457065,
          "longitude": -73.637993,
        },
        Object {
          "latitude": 45.456749,
          "longitude": -73.638252,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.45669,
          "longitude": -73.636975,
        },
        Object {
          "latitude": 45.456954,
          "longitude": -73.636774,
        },
        Object {
          "latitude": 45.457289,
          "longitude": -73.637637,
        },
        Object {
          "latitude": 45.457022,
          "longitude": -73.637836,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
  <AIRMapPolygon
    coordinates={
      Array [
        Object {
          "latitude": 45.456947,
          "longitude": -73.640738,
        },
        Object {
          "latitude": 45.457184,
          "longitude": -73.640564,
        },
        Object {
          "latitude": 45.457044,
          "longitude": -73.640165,
        },
        Object {
          "latitude": 45.456801,
          "longitude": -73.640348,
        },
      ]
    }
    fillColor={null}
    onPress={[Function]}
    strokeColor="#000"
    strokeWidth={1}
    tappable={true}
  />
</View>
`;
